tiledb/bioimg/converters/ome_tiff.py

```diff
@@ -274,17 +286,86 @@ def __init__(self, output_path: str):
         self._output_path = output_path
 
     def write_group_metadata(self, metadata: Mapping[str, Any]) -> None:
-        tiffwriter_kwargs = json.loads(metadata["json_tiffwriter_kwargs"])
-        tiffwriter_kwargs.pop("append")
         self._writer = tifffile.TiffWriter(
-            self._output_path, shaped=False, append=False, **tiffwriter_kwargs
+            self._output_path, shaped=False, bigtiff=True, append=False, ome=True
```

Comment:
Why to overwrite or bypass the `group_metadata`? Aren't these values being read by the `tiff` itself? Why should we hardcode them?

Reply:
When writing the tiff file we can't estimate the final size whether it will surpass 4GBs so the `bigtiff` is set to true as a foolproof solution. The `ome` is set to true in order to provide a uniform export format from a any input format.

Reply:
Gotcha then I am more inclined into hardcoding these values in `group_metadata` property.  Let's dry-fy it.